### PR TITLE
Add Download button to case study

### LINKED ISSUE #4416: [Site] Download Case Study Button
### Current Behavior
We don't support downloading the Case Study or any of our resource articles to be downloaded as pdf.

### Desired Behavior
Add a Download case study button on the resource article only if the type is a case study. 

### Implementation
Add it in - https://layer5.io/resources/cloud-native/hpes-adoption-of-meshery-and-meshmap, and add the button under the introduction.

### Acceptance Tests
The button should be a reusable component and should work according to the tag. 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Mohith234**
**Description**

This PR fixes #4416 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d407a6c25e9c4a8567ad81fa43bb13bd1ffdf0a4 at: https://64c1294d49440a0e5e31da9f--layer5.netlify.app

**Review by Chadha93 — COMMENTED** [review]
@Mohith234 I'm getting this error on download 
image

**Comment by l5io** [comment]
🚀 Preview for commit 228f98df755827a0e0f0927dd426d3d95d7596c7 at: https://64c20d27c17aa14fd056bf09--layer5.netlify.app

**Comment by DkRadhey** [comment]
@Mohith234 I'm not able to download the file.
image

**Comment by leecalcote** [comment]
I created a new PDF and changed the button.

File: src/collections/resources/case-study/hpes-adoption-of-meshery-and-meshmap/index.mdx [hunk 1]
```diff
@@ -35,6 +35,13 @@ import Yogi from "./yogi.webp"
 HPE's adoption of Meshery was driven by the need to simplify Kubernetes cluster management and monitoring. Meshery is an open source cloud native management tool that provides a self-service platform for designing, visualizing, deploying, testing, and operating cloud native infrastructure.
 </HPEintro>
 
+<div style={{display: "flex", justifyContent: "center", padding: "2%"}}>
+<a href="/assets/case-study/HPE's_adoption_of_Meshery_and_Meshmap.pdf" download="HPE's_adoption_of_Meshery_and_Meshmap.pdf">
+   <Button secondary title="Download Case Study" external={true}>
+   </Button>
+</a>
+</div>
+
```
Comment (leecalcote) [inline src/collections/resources/case-study/hpes-adoption-of-meshery-and-meshmap/index.mdx:44]:
```suggestion
<div style={{display: "flex", justifyContent: "center", padding: "2%"}}>

   <Button secondary title="Download Case Study" alt="Download HPE's Adoption of Layer5 Meshery and MeshMap" url="/case-studies/hpe-adoption-of-layer5-meshery-and-meshmap.pdf" />

</div>

```

**Comment by leecalcote** [comment]
Thanks for your help, @Mohith234

**Comment by Mohith234** [comment]
Thanks for the fix @leecalcote 🙌

**Comment by leecalcote** [comment]
Teamwork. Thanks for jumping in and helping, @Mohith234. 👍

**Pull request merged**